packages-next/admin-ui/src/utils/useLazyMetadata.tsx

```diff
@@ -27,7 +27,7 @@ export function useLazyMetadata(
   visibleLists: VisibleLists;
   createViewFieldModes: CreateViewFieldModes;
 } {
-  let result = useQuery(query, { errorPolicy: 'all' });
+  let result = useQuery(query, { errorPolicy: 'all', fetchPolicy: 'network-only' });
```

Comment:
I don't really understand why caching this breaks things but not caching this is basically totally fine because this will never be unmounted so the cache isn't doing anything